a3p-integration/proposals/a:upgrade-next/exit-reclaim.test.js

```diff
@@ -0,0 +1,61 @@
+import test from 'ava';
+import { execFile } from 'node:child_process';
+import { promisify } from 'node:util';
+
+const offerId = 'bad-invitation-15'; // cf. prepare.sh
+const from = 'gov1';
+
+test.before(async t => {
+  const $ = async (file, args) => {
```

Comment:
Why does this have a name which is a symbol rather than a descriptive string?

Reply:
it's patterned after zx and execa. maybe not a good enough reason

Reply:
ok if I leave it?

Reply:
not a blocker but I'd prefer we import `execa` instead of approximating it, so that it works consistently with all the other uses of it.

I also don't think we need to confine this helper to `context`. it's a test

Reply:
> prefer we import `execa`

ok.

I had just finished the `exitOffer.js` CLI tool where a single file that uses only node's standard library would simplify installation / deployment and my mind was still in that mode.